### DISCUSSION TITLE
Can't sign into my personal account

### DISCUSSION THREAD

**Discussion by rizkisahat996** [vercel/community#4523] [Account Issue] [answered]
### Question

Vercel keeps sign me into my teams account, when I try to sign into my personal account it return error 404 and ask to use different account

Please Help me cause one of my domain is stuck in there and I can't configure it 🙏

**Comment by amyegan** [comment, marked as answer]
Sorry you're having trouble with your account! We can't safely discuss account information in public because of security concerns. 

To create a ticket, please use the appropriate [direct channel](https://vercel.com/guides/how-to-get-vercel-support#direct-channels) found in this guide: [What is the best way to get support from Vercel?](https://vercel.com/guides/how-to-get-vercel-support#direct-channels)

If you let me know your ticket number, I can check on it for you 🙇

**Reply by OliverRC** [reply]
How can you link to the support request channel when selecting "Account Management" as the Problem Area redirects you to this Github community

**Reply by amyegan** [reply]
Sorry for the confusion! The support form is not the only direct channel listed in that guide 🙂 

> If you are unable to log in or experience difficulties during registration, please email [registration@vercel.com](mailto:registration@vercel.com) and include the details of the project you are looking to deploy with Vercel.
